conf/deployment/vsphere/upi_1az_rhcos_vsan_3m_3w_custom_storageclass_names.yaml

```diff
@@ -0,0 +1,25 @@
+---
+# This config is suppose to work on most of DCs we have.
```

Comment:
I would mention here the use case of the custom storage classes.

Reply:
Done